src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -712,6 +712,12 @@ Prevent Long URL From Breaking Out of Container
     hyphens: auto;
 }
 
+/* highlight search keywords in result on Admin search page */
+.highlight {
+        background-color: #FFFF88;
+    }
```

Comment:
Please fix the indentation. I'm not OCD but we need follow the standard code style.